src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDatabaseMetaData.java

```diff
@@ -244,12 +244,10 @@ private SQLServerResultSet getResultSetFromInternalQueries(String catalog,
         SQLServerResultSet rs = null;
         try {
             rs = ((SQLServerStatement) connection.createStatement()).executeQueryInternal(query);
-
         }
         finally {
-            if (null != orgCat) {
+            if (null != orgCat)
```

Comment:
actually we started adding back the brackets even with one line of code, so lets keep the brackets here  😄